bot/cogs/cogs.py

```diff
@@ -0,0 +1,218 @@
+# coding=utf-8
+import os
+
+from discord import ClientException, Colour, Embed
+from discord.ext.commands import AutoShardedBot, Context, command
+
+from bot.constants import (
+    ADMIN_ROLE, BOT_AVATAR_URL, DEVOPS_ROLE, GITHUB_URL_BOT, GREEN_CHEVRON, MODERATOR_ROLE, OWNER_ROLE, RED_CHEVRON
+)
+from bot.decorators import with_role
+from bot.utils import paginate
+
+
+class Cogs:
+    """
+    Cog management commands
+    """
+
+    def __init__(self, bot: AutoShardedBot):
+        self.bot = bot
+        self.cogs = {}
+
+        # Load up the cog names
+        for filename in os.listdir("bot/cogs"):
+            if filename.endswith(".py") and "_" not in filename:
+                if os.path.isfile(f"bot/cogs/{filename}"):
+                    cog = filename[:-3]
+
+                    self.cogs[cog] = f"bot.cogs.{cog}"
+
+        # Allow reverse lookups by reversing the pairs
+        self.cogs.update({v: k for k, v in self.cogs.items()})
+
+    @command(name="cogs.load()", aliases=["cogs.load", "load_cog"])
+    @with_role(MODERATOR_ROLE, ADMIN_ROLE, OWNER_ROLE, DEVOPS_ROLE)
+    async def load_command(self, ctx: Context, cog: str):
```

Comment:
do the methods really have to be called x_command, can't they just all be called x? it's clear that they are commands from the decorator.

Reply:
I started doing that with ClickUp because I was shadowing builtins and reserved words

Reply:
okay. then I suppose I should do the same in tags?

Reply:
I'm not super worried about it, but I guess it's a good habit